vault/docs/redeem.md

```diff
@@ -0,0 +1,9 @@
+# Redeem Processes
+
+When the Vault observes a Redeem for it's locked BTC, it must execute the payment on Bitcoin and submit the transaction inclusion proof.
+
+<p align="center">
+  <img src="./img/redeem.png" alt="redeem" width="500"/>
```

Comment:
Maybe add a "wait until block is relayed and has sufficient bitcoin&parachain confirmations" block?